### DISCUSSION TITLE
Sigmac Output To Elastalert

### DISCUSSION THREAD

**Discussion by OmerTirosh** [Security-Onion-Solutions/securityonion#2746] [General] [answered]
Hi,

I have seen issue #89 and didn't understand whatever the sigmac output is elastalert or esquery now.

It seems that you can't change elastalert rule type or use aggregations. 

Can anyone explain why?

**Comment by weslambert** [comment, marked as answer]
The rules to which you are speaking are managed by Playbook, so it will manage them, and modifications to those rules will be overwritten.  You should, however, be able to tweak those rules from Playbook, as opposed to directly in the rule itself.
